cli/src/commands/tunnels.rs

```diff
@@ -74,7 +74,7 @@ impl ServiceContainer for TunnelServiceContainer {
 		&mut self,
 		log: log::Logger,
 		launcher_paths: LauncherPaths,
-		shutdown_rx: mpsc::Receiver<ShutdownSignal>,
+		shutdown_rx: mpsc::UnboundedReceiver<ShutdownSignal>,
```

Comment:
I changed this, since we need to send shutdown synchronously from the windows service. Bounded receivers require the caller to `.await` them, since they might need to wait for the receiver to process the data, but unbounded receivers can enqueue messages synchronously.